tee-worker/ts-tests/integration-tests/vc_correctness.test.ts

```diff
@@ -43,7 +43,7 @@ describe('Test Vc (direct invocation)', function () {
     const errorArray: { id: string; index: number; assertion: any; error: any }[] = [];
     this.timeout(6000000);
     before(async () => {
-        context = await initIntegrationTestContext(nodeEndpoint);
+        context = await initIntegrationTestContext(nodeEndpoint, enclaveEndpoint);
```

Comment:
There're more places using `initIntegrationTestContext` - do they need to be adjusted too, or only this test?

Reply:
Currently only vc correctness is testing tee-dev, the rest are ci staging tests, no need to change.